apps/shared/test-helpers/ganache-cli.sh

```diff
@@ -38,6 +38,7 @@ start_testrpc() {
 		aragon devchain --reset --port "$testrpc_port" &
 	elif [ "$DEV" = true ]; then
 		aragon devchain --reset --port "$testrpc_port" &
+		sleep 10 # wait for devchain to start TODO: modify cli to return rather than require interruption
```

Comment:
No the most efficient way to do, but if that works, for now should be enough...
Hopefully we will get rid of this whole script sooner rather than later

Reply:
Exactly! I added similar comments to the commit message where I did this.